### ISSUE TITLE
Double author name

### ISSUE THREAD

**Issue by peterdesmet** [inbo/mica-occurrences#22]
Kelly Meijvisch is included as contributor with two email addresses. @EmmaCartuyvels1 I assume this is the same person?

**Comment by sannegovaert** [comment]
This can be closed?

**Comment by peterdesmet** [comment]
Yes, the person is only listed once in the public dataset: https://www.gbif.org/dataset/8a5cbaec-2839-4471-9e1d-98df301095dd#contacts
